test/unit/annotation_layer_spec.js

```diff
@@ -518,6 +524,74 @@ describe('Annotation layer', function() {
       expect(textWidgetAnnotation.data.readOnly).toEqual(true);
       expect(textWidgetAnnotation.data.multiLine).toEqual(true);
     });
+
+    it('should reject comb fields without a maximum length', function() {
+      var flags = 0;
+      flags |= 1 << (AnnotationFieldFlag.COMB - 1);
+
+      textWidgetDict.set('Ff', flags);
+
+      var textWidgetRef = new Ref(46, 0);
+      var xref = new XRefMock([
+        { ref: textWidgetRef, data: textWidgetDict, }
+      ]);
+
+      var textWidgetAnnotation = annotationFactory.create(xref, textWidgetRef);
+      expect(textWidgetAnnotation.data.comb).toEqual(false);
+    });
+
+    it('should accept comb fields with a maximum length', function() {
+      var flags = 0;
+      flags |= 1 << (AnnotationFieldFlag.COMB - 1);
+
+      textWidgetDict.set('MaxLen', 20);
+      textWidgetDict.set('Ff', flags);
+
+      var textWidgetRef = new Ref(46, 0);
+      var xref = new XRefMock([
+        { ref: textWidgetRef, data: textWidgetDict, }
+      ]);
+
+      var textWidgetAnnotation = annotationFactory.create(xref, textWidgetRef);
+      expect(textWidgetAnnotation.data.comb).toEqual(true);
+    });
+
+    it('should only accept comb fields when the flags are valid', function() {
```

Comment:
It might be possible to simplify this particular test a bit, so I'd like to see an updated version based on the `AnnotationFieldFlag` comment above.